File: API/Features/Billing/Prices/Dtos/PriceWriteDto.cs
using System;
using API.Infrastructure.Interfaces;

namespace API.Features.Billing.Prices {

    public class PriceWriteDto : IMetadata {

        // PK
        public Guid Id { get; set; }
        // FKs
        public int CustomerId { get; set; }
        public int DestinationId { get; set; }
        public int PortId { get; set; }
        // Fields
        public string From { get; set; }
        public string To { get; set; }
        public decimal AdultsWithTransfer { get; set; }
        public decimal AdultsWithoutTransfer { get; set; }
        public decimal KidsWithTransfer { get; set; }
        public decimal KidsWithoutTransfer { get; set; }
        // Metadata
        public string PostAt { get; set; }
        public string PostUser { get; set; }
        public string PutAt { get; set; }
        public string PutUser { get; set; }

    }

}

File: FrontEnd/src/app/features/billing/prices/classes/dtos/price-write-dto.ts
import { Guid } from 'guid-typescript'

export interface PriceWriteDto {

    // PK
    id: Guid
    // FKs
    customerId: number
    destinationId: number
    portId: number
    // Fields
    from: string
    to: string
    adultsWithTransfer: number
    adultsWithoutTransfer: number
    kidsWithTransfer: number
    kidsWithoutTransfer: number
    // Rowversion
    putAt: string

}
